### PR TITLE
Update Configuring a codespace for VS

### PR DESCRIPTION
**Pull request by gregvanl**
### Why:

Visual Studio devinit tool team requested changes to the section on Configuring a codespace for VS

### What's being changed:

See PR for content changes

### Check off the following:
- [x] All of the tests are passing.
- [x] I have reviewed my changes in staging. (look for the **deploy-to-heroku** link in your pull request, then click **View deployment**)
- [x] For content changes, I have reviewed the [localization checklist](https://github.com/github/docs/blob/main/contributing/localization-checklist.md)
- [x] For content changes, I have reviewed the [Content style guide for GitHub Docs](https://github.com/github/docs/blob/main/contributing/content-style-guide.md).

### CODE REVIEW

**Comment by janiceilene** [comment]
@gregvanl Thanks so much for opening a PR! I'll get this triaged for review ✨

File: content/github/developing-online-with-codespaces/using-codespaces-in-visual-studio.md [hunk 1]
```diff
@@ -53,8 +53,12 @@ Connecting to {% data variables.product.prodname_github_codespaces %}  with the
...
+[devinit](https://docs.microsoft.com/visualstudio/devinit/getting-started-with-devinit) lets you install additional frameworks and tools into your Windows development codespaces, modify environment variables, and more. 
+
+devinit supports a configuration file called [devinit.json](https://docs.microsoft.com/visualstudio/devinit/devinit-json), which can be added to your project for creating customized and repeatable development environments. Used with a `.devcontainer.json`, your codespace will be automatically configured upon creation.
```
Comment (hubwriter) [inline content/github/developing-online-with-codespaces/using-codespaces-in-visual-studio.md:62]: Small change to use active voice. 
I also wondered what  "Used with a `.devcontainer.json`" meant when I read this. It's explained in the page referenced in the sentence that follows this, but at this stage readers may not know. Perhaps link to the relevant section of that page here:

```suggestion
devinit supports a configuration file called [devinit.json](https://docs.microsoft.com/visualstudio/devinit/devinit-json). You can add this to your project if you want to create a customized and repeatable development environment. Used with a `.devcontainer.json`, your codespace will be automatically configured on creation.
```

https://docs.microsoft.com/en-us/visualstudio/ide/codespaces/customize-codespaces?view=vs-2019#running-devinit-when-creating-a-codespace

**Review by hubwriter — COMMENTED** [review]
@gregvanl - Many thanks for submitting this PR. I've made a small suggestion for a change. Thanks again! 👍

File: content/github/developing-online-with-codespaces/using-codespaces-in-visual-studio.md (hunk 1 above)
Comment (gregvanl) [reply content/github/developing-online-with-codespaces/using-codespaces-in-visual-studio.md]: Good suggestions. I pushed a commit with a few additional changes and a normalized URL.

**Comment by hubwriter** [comment]
@gregvanl - Thanks again for the update to this topic. Merging now.

**Review by hubwriter — APPROVED** [review]

**Pull request merged**